app/initializers/irene.js

```diff
@@ -16,6 +16,40 @@ const initialize = function(application) {
   // Inject Store
   application.inject('component', 'store', 'service:store');
 
+  let runtimeConfig = window.runtimeGlobalConfig;
+
+  function parseUrl(url) {
+    let parser = document.createElement('a'), searchObject = {}, queries, split, i;
```

Comment:
instead of a tag can you use the webapi https://developer.mozilla.org/en-US/docs/Web/API/URL? this api is available in node also natively.

Reply:
done